src/app/containers/Home/index.js

```diff
@@ -3,61 +3,171 @@ import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { bindActionCreators } from 'redux'
 import { withRouter } from 'react-router-dom'
+import { getAccountName, validateLength } from '../../utils/helpers'
 
 import Neon from '@cityofzion/neon-js'
-
-import { Button } from 'rmwc/Button'
-import '@material/button/dist/mdc.button.min.css'
-
 import withLoginCheck from '../../components/Login/withLoginCheck'
 
-import style from './Home.css'
+import Home from './Home'
 
 import * as AccountActions from '../../actions/account'
+import * as walletActions from '../../actions/wallet'
 
 @connect(
   state => ({
     account: state.account,
+    accounts: state.wallet.accounts,
+    selectedNetworkId: state.config.selectedNetworkId,
   }),
   dispatch => ({
     actions: bindActionCreators(AccountActions, dispatch),
+    walletActions: bindActionCreators(walletActions, dispatch),
   })
 )
+class HomeContainer extends Component {
```

Comment:
Generally we index.js to wrap the component of the directory's name (in this case Home). Mean it passes in the redux state etc. It's strange to setup a component with a different name in index.js imo.